### PR TITLE
7213 claims status counting

### PR DESCRIPTION
**Pull request by kpethtel**
## Summary
Adds the new `activeClaims` field to claims and appeals met. Claims are considered active when they are not complete. 

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va-mobile-app/issues/7213

## Testing done

- Specs

## What areas of the site does it impact?
Adds to claims and appeals meta but should not impact the endpoint otherwise. Will test in staging.

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

File: modules/mobile/app/services/mobile/v0/claims/proxy.rb [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-require 'lighthouse/facilities/client'
```
Comment (kpethtel) [inline modules/mobile/app/services/mobile/v0/claims/proxy.rb:3]: I believe we refactored this a few months ago and this is no longer needed in this file.

**Review by aherzberg — APPROVED** [review]
LGTM

**Review by aherzberg — APPROVED** [review]
LGTM

**Review by aherzberg — APPROVED** [review]
LGTM

**Pull request merged**